### PR TITLE
Add authorizer log config. 

### PR DESCRIPTION
**Pull request by saheemg**
@johnbellone

### CODE REVIEW

File: libraries/kafka_config.rb [hunk 1]
```diff
@@ -34,7 +34,8 @@ class KafkaConfig < Chef::Resource
                     'kafka.request.logger' => 'WARN',
                     'kafka.controller' => 'TRACE',
                     'kafka.log.LogCleaner' => 'INFO',
-                    'state.change.logger' => 'TRACE'
+                    'state.change.logger' => 'TRACE',
+                    'kafka.authorizer.logger' => 'WARN'
```
Comment (johnbellone) [inline libraries/kafka_config.rb]: I am not opposed to this change, but for my own understanding, this is logger new to a specific version of Kafka? If so, we should consider making that the minimum version of the cookbook.

Comment (saheemg) [inline libraries/kafka_config.rb]: I believe this logger is as of 0.9 of kafka.  Shouldn't it be a no-op if its an older version of kafka?

Comment (johnbellone) [inline libraries/kafka_config.rb]: Do you know that for sure? If you don't let's just bump the minimum version :).

Comment (saheemg) [inline libraries/kafka_config.rb]: do you want me to update https://github.com/bloomberg/kafka-cookbook/blob/master/attributes/default.rb#L44 ?  That means I need to update the checksum on https://github.com/bloomberg/kafka-cookbook/blob/master/attributes/default.rb#L46, right?  

But it looks like the mirror defined on https://github.com/bloomberg/kafka-cookbook/blob/master/attributes/default.rb#L47 no longer works.

**Comment by saheemg** [comment]
@johnbellone ping

**Comment by johnbellone** [comment]
Good point, I'll merge this and fix it!

**Pull request merged**